src/test/test_runner.py

```diff
@@ -126,6 +126,7 @@ def test_prepare_environment_with_git(self):
                 "GIT_BRANCH": conf.git_branch,
                 "GIT_SHA_SHORT": conf.git_sha_short,
                 "GIT_REMOTE_ORIGIN_URL": conf.git_remote_origin_url,
+                "GIT_TAG": conf.git_tag,
```

Comment:
we also need to tag the HEAD of the branch we created, and then check that that's what we're getting back